tests/acceptance/shared/util/constants.go

```diff
@@ -0,0 +1,85 @@
+package util
+
+import (
+	"flag"
+
+	flag2 "github.com/k3s-io/k3s/tests/acceptance/core/service/flag"
+)
+
+var (
+	Destroy        = flag.Bool("destroy", false, "a bool")
+	UpgradeVersion = flag.String("upgradeVersion", "", "Version to upgrade the cluster to")
+	Arch           = flag.String("arch", "amd64", "a string")
+
+	ExpectedValueUpgradedHost string
+	ExpectedValueUpgradedNode string
+	CmdHost                   string
+	ExpectedValueHost         string
+	CmdNode                   string
+	ExpectedValueNode         string
+	Description               string
+	InstallUpgradeFlag        flag2.MultiValueFlag
+	TestCase                  flag2.TestConfigFlag
```

Comment:
Several of these flags seem to specific to be in a utility library, such as `ExpectedValueUpgradedHost` and `InstallUpgradeFlag`. I would recommend removing them from the package.